### PR TITLE
Move play / pause buttons to the side of animated time-series

### PR DESCRIPTION
**Pull request by rjimenezda**
re: https://github.com/CartoDB/cartodb/issues/12387

---

Cartodb PR: https://github.com/CartoDB/cartodb/pull/12482
Deep-insights.js PR: https://github.com/CartoDB/deep-insights.js/pull/561
Cartodb.js PR: https://github.com/CartoDB/cartodb.js/pull/1701

---

Play button is here now:

screen shot 2017-07-17 at 12 34 19

It also does most of the job of enabling playback on a selected range, but it's not perfect on a couple of cases, so i've disabled it for now. The cartodb.js work is on the branch 12387-play-pause, which we don't need to merge ASAP.

### CODE REVIEW

File: src/widgets/histogram/chart.js [hunk 1]
```diff
@@ -124,14 +119,38 @@ module.exports = cdb.core.View.extend({
...
       var wasHidden = this.isHidden();
 
       this.hide();
 
-      var width = this.$el.parent().width() || 0;
+      var parent = this.$el.parent();
+      var width = parent.width() || 0;
+
+      if (this.model.get('animated')) {
+        // We could just substract 24, width of play/pause but imho this is more future proof
```
Comment (matallo) [inline src/widgets/histogram/chart.js]: 👍

File: src/widgets/time-series/torque-time-slider-view.js [hunk 2]
```diff
@@ -60,9 +61,9 @@ module.exports = cdb.core.View.extend({
     this.add_related_model(this._chartView.model);
 
     this._dataviewModel.on('change:bins', this._updateChartandTimeslider, this);
-    this.add_related_model(this._dataviewModel);
-
     this._dataviewModel.filter.on('change:min change:max', this._onFilterMinMaxChange, this);
+
+    this.add_related_model(this._dataviewModel);
```
Comment (matallo) [inline src/widgets/time-series/torque-time-slider-view.js]: why not moving to `listenTo` and get rid of related models?

File: themes/scss/widgets/themes/_light.scss [hunk 3]
```diff
@@ -61,13 +61,13 @@ $timeSliderBorder: rgba(#FFF, 1);
...
   }
-  .CDB-Widget-controlButtonIcon--play {
-    border-left-color: $elementBkg;
-  }
-  .CDB-Widget-controlButtonIcon--pause {
-    &:before,
-    &:after {
-      background-color: $elementBkg;
+
+  .CDB-Widget-controlButton--disabled {
```
Comment (matallo) [inline themes/scss/widgets/themes/_light.scss]: being nitpicking here, disabled should be a `.is-state` class, not a `.*--modifier`

File: src/widgets/time-series/torque-time-slider-view.js (hunk 2 above)
Comment (xavijam) [inline src/widgets/time-series/torque-time-slider-view.js]: Agree :+1:, I'll take care of it 💋

File: themes/scss/widgets/themes/_light.scss (hunk 3 above)
Comment (xavijam) [inline themes/scss/widgets/themes/_light.scss]: Agree too.

**Comment by xavijam** [comment]
CR feedback applied @matallo 💋

**Comment by xavijam** [comment]
Our CI passing, merging!

**Pull request merged**